h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -3219,7 +3219,13 @@ def model_seed_sorted(model_list):
             if model_list.models[index]._model_json["parameters"][pIndex]["name"]=="seed":
                 model_seed_list[index]=model_list.models[index]._model_json["parameters"][pIndex]["actual_value"]
                 break
+    print("model seed list before sort: ")
```

Comment:
these look like debugging printouts, do we want to keep those?

Reply:
no, we don't, there is a separate JIRA for this issue. will remove this